src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -135,47 +149,46 @@ export const RegistrationFormActions = ({
           onClick={toggleSupportModal}>
           {t('my_page.messages.get_curator_support')}
         </Button>
-        {!isLastTab ? (
-          <Box
-            sx={{
-              gridArea: 'save-button',
-              display: 'flex',
-              justifyContent: 'end',
-              alignItems: 'center',
-            }}>
+
+        <Box
+          sx={{
+            gridArea: 'action-buttons',
+            display: 'flex',
+            justifyContent: 'end',
+            alignItems: 'center',
+            gap: '0.5rem',
+          }}>
+          <Button data-testid={dataTestId.registrationWizard.formActions.cancelEditButton} onClick={cancelEdit}>
+            {t('common.cancel')}
+          </Button>
+          {!isLastTab ? (
             <LoadingButton
               variant="outlined"
               loading={isSaving}
               data-testid={dataTestId.registrationWizard.formActions.saveRegistrationButton}
               onClick={handleSaveClick}>
               {t('common.save')}
             </LoadingButton>
-            <Tooltip title={t('common.next')} sx={{ gridArea: 'next-button' }}>
+          ) : (
+            <LoadingButton
+              variant="contained"
+              loading={isSaving}
+              data-testid={dataTestId.registrationWizard.formActions.saveRegistrationButton}
+              onClick={handleSaveClick}>
+              {t('common.save_and_view')}
+            </LoadingButton>
+          )}
+          <Tooltip title={t('common.next')}>
+            <span>
```

Comment:
Ulempen med å vise knappene inkl. tooltip også når de er disabled er at det kan virke som en bug, siden man ikke kan komme til neste uansett akkurat i dette tilfellet. Bare lufter det, men har ingen sterke meninger her, og vet ikke hva som er vanlig/best practice for sånt

![bilde](https://github.com/user-attachments/assets/95920987-c637-4f32-bf91-1321f8f48405)

Reply:
Gikk for å ikke ha `Tooltip` når den er disabled 👍

Reply:
da er vel denne span-en overflødig igjen?